backend/routes/graph.py

```diff
@@ -42,84 +36,98 @@
-        # TODO: needs to be the list of missing nodes with vocab and nonstandard id. probably is get_concepts()
-        missing_in_betweens = ''
-        concept_metadata_m, hidden_dict_m, nonstandard_concepts_hidden_m = filter_concepts(missing_in_betweens)
-
-        # TODO: merge/update the 3 variables each from the 2 returns of filter_concepts()
-        concept_ids.update(missing_in_betweens_ids)
-
-        # Get subgaph
-        sg = REL_GRAPH.subgraph(concept_ids)
-
+        await rpt.start_rpt(request, params={'codeset_ids': codeset_ids, 'extra_concept_ids': extra_concept_ids})
+        sg, missing_in_betweens, hidden_dict, nonstandard_concepts_hidden = await concept_graph(
+            request, codeset_ids, extra_concept_ids, hide_vocabs, hide_nonstandard_concepts, verbose)
         await rpt.finish(rows=len(sg))
-        VERBOSE and timer('done')
+        # TODO: see return for concept_graph() and add anything missing bleow
+        return {'edges': list(sg.edges), 'filled_gaps': missing_in_betweens}
     except Exception as e:
         await rpt.log_error(e)
         raise e
+
+
+# TODO: Joe refactor: after get_concepts_for_graph()
+#  - update typedefs returns here and for concept_graph()
+# TODO: Now we're calling missing-in-between-nodes. Do we want to do that before or after filtering (for voc and
+#  nonstandard concepts)?
+# TODO: use hide_vocabs()
+async def concept_graph(
+    codeset_ids: List[int], extra_concept_ids: Union[List[int], None] = [], hide_vocabs = ['RxNorm Extension'],
+    hide_nonstandard_concepts=False, verbose = VERBOSE
+) -> List:
+    """Return concept graph
+
+    :param extra_concept_ids: Not used right now. This is for when we allow user to add concepts from other concept sets
+    . They're not part of any codesets yet, as these are added by user.
+    :returns
+      hidden_dict: Map of vocab to set of concept ids"""
+    if extra_concept_ids:
+        raise NotImplementedError('See docstring for concept_graph()')
+    # todo: old code to remove. Returns nothing right now. all code below this point in concept_graph_post() used to
+    #  be part of get_connected_subgraph() or one of its sub-functions.
+    # (nodes_in_graph, missing_in_between_nodes, preferred_concept_ids, orphans_not_in_graph, hidden_nodes,
+    #  hidden_dict) = get_connected_subgraph(REL_GRAPH, codeset_ids, extra_concept_ids, hide_vocabs=['RxNorm Extension'])
+    # tree = await indented_concept_list(codeset_ids, extra_concept_ids, hide_vocabs)
+    # sg, filled_gaps = fill_in_gaps(REL_GRAPH, id, return_missing=True)
+    timer = get_timer('')
+    verbose and timer('concept_graph()')
+
+    # Get concepts & metadata
+    concepts_raw: List[Dict] = [dict(x) for x in get_cset_members_items(
+        codeset_ids=codeset_ids, columns=['concept_id', 'vocabulary_id', 'standard_concept'])]
+    # Filter
+    # TODO: see: filter_concepts()
+    concepts: List[Dict]
+    hidden_dict: Dict[str, Set[int]]
```

Comment:
## Question: `hidden_by_voc`
Formerly `hidden_dict`. It actually has been not a set of ints, but a set of dicts. Kept it that way. Is that correct?